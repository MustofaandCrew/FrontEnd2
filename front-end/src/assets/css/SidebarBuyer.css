
.logo {
    margin: 0 auto;
    align-items: center;
    justify-content: center;
}
.sidebar {
    width: 250px !important;
    height: 100%;
    background-color: #fff;
    transition: 0.3s;
}
.link {
    width: 100%;
    padding: 7.5px 0;
}
.link:hover {
    background-image: linear-gradient(90deg, #fff 0%, #FF7158  150%);
}
.foto-profil{
    width: 35px;
    border-radius: 20px;
    margin-right: 10px;
}
.text-left {
    text-align: left;
    margin-left: -40px
}
.text-right {
    text-align: right;
}
