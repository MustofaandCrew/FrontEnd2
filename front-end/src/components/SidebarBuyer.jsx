import React from 'react';
import {
  CDBSidebar,
  CDBSidebarContent,
  CDBSidebarFooter,
  CDBSidebarHeader,
  CDBSidebarMenu,
  CDBSidebarMenuItem,
} from 'cdbreact';
import { NavLink } from 'react-router-dom';
import '../assets/css/SidebarBuyer.css'

const SidebarBuyer = () => {
  return (
    <div
      className='main'
    >
      <CDBSidebar className='sidebar'>
        <CDBSidebarHeader>
          <a
            href="/"
            className="text-decoration-none"
          >
            Sidebar
          </a>
        </CDBSidebarHeader>

        <CDBSidebarContent className="sidebar-content">
          <CDBSidebarMenu>
            <NavLink exact to="/transaksibuyer" activeClassName="activeClicked">
              <CDBSidebarMenuItem icon="columns">Dashboard</CDBSidebarMenuItem>
            </NavLink>
            <NavLink exact to="/accountbuyer" activeClassName="activeClicked">
              <CDBSidebarMenuItem icon="table">My Account</CDBSidebarMenuItem>
            </NavLink>

            <NavLink
              exact
              to="/hero404"
              target="_blank"
              activeClassName="activeClicked"
            >
              <CDBSidebarMenuItem icon="exclamation-circle">
                404 page
              </CDBSidebarMenuItem>
            </NavLink>
          </CDBSidebarMenu>
        </CDBSidebarContent>

        <CDBSidebarFooter className="footer">
          <div
            className='footer-text'
          >
            Sidebar Footer
          </div>
        </CDBSidebarFooter>
      </CDBSidebar>
    </div>
  );
};

export default SidebarBuyer;