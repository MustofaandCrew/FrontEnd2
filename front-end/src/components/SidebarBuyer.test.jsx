import React from "react";
import renderer from "react-test-renderer";
import SidebarBuyer from "../components/SidebarBuyer";


describe("Test suits for SidebarBuyer", () => {
    it("should match with snapshot", () => {
        const tree = renderer.create(<SidebarBuyer />).toJSON();
    });
});