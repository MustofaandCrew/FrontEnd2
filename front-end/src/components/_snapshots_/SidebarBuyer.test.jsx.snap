// Jest Snapshot v1, https://goo.gl/fbAQLP
import SidebarBuyer from "../components/SidebarBuyer";

exports["SidebarBuyer"] = {
    test: function test() {
        var tree = renderer.create(<SidebarBuyer />).toJSON();
        expect(tree).toMatchSnapshot();
    }
};